apis/fluent/atrium-api-fluent/src/commonMain/kotlin/ch/tutteli/atrium/api/fluent/en_GB/listExpectations.kt

```diff
@@ -0,0 +1,20 @@
+package ch.tutteli.atrium.api.fluent.en_GB
+
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.logic._logic
+import ch.tutteli.atrium.logic.last
+
+/**
+ * Expects that the subject of `this` expectation (a [List]) is not empty and that the last element
+ * in the list holds all assertions the given [IndexWithCreator.assertionCreator] creates for it.
+ *
+ *  Use the function `last() { ... }` to create an [IndexWithCreator].
```

Comment:
Also this sentence if from the api-infix and is not necessary here as one does not need to pass an IndexWithCreator
```suggestion
```